Classes/Emogrifier.php

```diff
@@ -411,6 +432,32 @@ public function removeUnprocessableHtmlTag($tagName)
             unset($this->unprocessableHtmlTags[$key]);
         }
     }
+    
+    /**
+     * Adds a selector to exclude nodes from processing
+     *
+     * Any nodes that match the selector will not have their style altered.
+     *
+     * @param string $selector the selector to exclude, e.g., ".editor"
+     *
+     * @return void
+     */
+    public function addExcludedSelector($selector)
+    {
+        $this->excludedSelectors[] = $selector;
```

Comment:
I propose you use the same approach as with $this->allowedMediaTypes. This makes avoiding duplicates and removing entries a lot simpler.